src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -94,6 +94,9 @@ public String getType() {
     @Column(name = "allow_offline_ide", table = "programming_exercise_details")
     private Boolean allowOfflineIde;
 
+    @Column(name = "allow_online_ide", table = "programming_exercise_details")
+    private Boolean allowOnlineIde;
```

Comment:
I know that other attributes also use `Boolean` instead of `boolean`, but is there a need here for the new attribute? Otherwise I would prefer the base type

Reply:
Related issue #6079.

Probably a default value of `false` would make sense.

Reply:
Very good aspects from both of you, thanks! Will incorporate both :)

Reply:
@b-fein the default value of `false` is actually already implemented in the DB. For new ProgrammingExercises, the default value (also `false`) is set in the constructor of the respective client component.